snoop/emails.py

```diff
@@ -95,6 +96,12 @@ def open_part(self, number):
             data = part.get_payload(decode=True)
         except:
             raise PayloadError
+
+        if "encrypted" in self.flags and \
```

Comment:
Oh, I completely forgot about the flags logic. I don't think it's being used for anything at the moment. I'd rather just kill it. And you can set `self.encrypted = True|False`.